site/source/docs/contributing/developers_guide.rst

```diff
@@ -37,36 +58,43 @@ When submitting patches, please:
 
 - Make pull requests to **incoming**, not master.
 - Do not include merge commits in pull requests; include only commits with the new relevant code.
```

Comment:
You could mention that our default github merge method is "squash and merge". Or maybe that makes this statement unnecessary? not sure.

Reply:
Oh right, this is out of date given we squash+merge - it's ok to have merge commits given that. I reworded and applied the other feedback.